pkg/BUILD.bazel

```diff
@@ -684,6 +684,7 @@ GO_TARGETS = [
     "//pkg/ccl/backupccl/backuprand:backuprand_test",
     "//pkg/ccl/backupccl/backupresolver:backupresolver",
     "//pkg/ccl/backupccl/backupresolver:backupresolver_test",
+    "//pkg/ccl/backupccl/backuptestutils:backuptestutils",
```

Comment:
Nit: I think you need `dev generate bazel` one more time to pull your new test into the list of all tests.